### ISSUE TITLE
Unable to use jsonpath with browserify

### ISSUE THREAD

**Issue by EarthyOrange** [dchester/jsonpath#166]
<pre>
[13:59:51] Browserify error: Can't walk dependency graph: Cannot find module '..' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
[13:59:51] Browserify error: Can't walk dependency graph: Cannot find module './dict' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
[13:59:51] Browserify error: Can't walk dependency graph: Cannot find module './aesprim' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
[13:59:51] Browserify error: Can't walk dependency graph: Cannot find module './slice' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
[13:59:51] Browserify error: Can't walk dependency graph: Cannot find module './handlers' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
[13:59:51] Browserify error: Can't walk dependency graph: Cannot find module './grammar' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
[13:59:51] Browserify error: Can't walk dependency graph: Cannot find module './parser' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
[13:59:52] Browserify error: Can't walk dependency graph: Cannot find module './support/isBuffer' from '.../node_modules/jsonpath'
    required by .../node_modules/jsonpath/jsonpath.js
</pre>

I ran in to an issue with browserify and jsonpath as well. However, the issue seems to be that the paths to modules in the require statments in the jsonpath.js file (https://github.com/dchester/jsonpath/blob/master/jsonpath.js#L5117) seem to be incorrect.

**Comment by carlosjrtee** [comment]
Hi @EarthyOrange ! I'm also having this same issue. Did you manage to find a workaround for it?

**Comment by EarthyOrange** [comment]
@carlosjrtee No, I haven't yet. Did you manage to resolve it in the meantime?
